app/api/v1/views/parcel_views.py

```diff
@@ -7,67 +7,96 @@
 
 # Multiple parcels class
 class Parcels(Resource):
+    def __init__(self):
+        self.user_id = 1
 
     # the user creates an order
     def post(self):
         data = request.get_json(force=True)
+        if "price" not in data:
+            return make_response(jsonify(message="incomplete request"), 400)
+        if "name" not in data:
```

Comment:
I think this can be improved to use one if condition since the response is the same.